std/algorithm/sorting.d

```diff
@@ -2918,6 +2963,40 @@ private template TimSortImpl(alias pred, R)
     sort!("a < b", SwapStrategy.stable)(arr);
 }
 
+@safe unittest
+{
+    static struct Postblit
```

Comment:
Since this uses a copy ctor and not postblit, maybe it shouldn't be called `Postblit`? ;)

Reply:
Fixed